addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
If you don't want to break those python addons, drop this silly "backwards-compatibility" statement in the xml. It is worth nothing if you can't rely on it. 

> EDIT: and hey, good written python addons are not supposed to crash. they are supposed to guard against such error. and they can. most of the time.

you are even saying that this is not needed. if it is a good addon it just does not need this info.